_assets/ci/Jenkinsfile-manual

```diff
@@ -71,32 +71,18 @@ node('linux') {
 
     stage('Deploy') {
         dir(cloneDir) {
-            // For branch builds, replace the old artifact. For develop keep all of them.
-            def version = sh(
-                script: "git describe --exact-match --tag 2>/dev/null || git describe --always",
-                returnStdout: true
-            ).trim()
-            def server = Artifactory.server 'artifacts'
-            def uploadSpec = """{
-                "files": [
-                    {
-                        "pattern": "build/bin/statusgo-android-16.aar",
-                        "target": "libs-release-local/status-im/status-go/${version}/status-go-${version}.aar"
-                    },
-                    {
-                        "pattern": "build/bin/statusgo-ios-9.3-framework/status-go-ios.zip",
-                        "target": "libs-release-local/status-im/status-go-ios-simulator/${version}/status-go-ios-simulator-${version}.zip"
-                    }
-                ]
-            }"""
-
-            def buildInfo = Artifactory.newBuildInfo()
-            buildInfo.env.capture = false
-            buildInfo.name = 'status-go (' + gitBranch + '-' + gitShortSHA + ')'
-            server.upload(uploadSpec, buildInfo)
-            server.publishBuildInfo(buildInfo)
-
-            slackSend color: 'good', message: 'status-go `' + version + '` was built successfully. ' + env.BUILD_URL
+            sh "make prepare-release"
+            withCredentials([[
+                $class: 'UsernamePasswordMultiBinding',
+                credentialsId: 'status-im-auto',
+                usernameVariable: 'GITHUB_USER',
+                passwordVariable: 'GITHUB_TOKEN'
+            ]]) {
+              sh """
+                yes | make release release_branch=${gitBranch}
```

Comment:
Is it gonna work if we have `if [ $$REPLY = "y" ];`? Also does it work without `echo` before `yes`?

Reply:
seems like, tested on jenkins already (`yes` should output `y`, at least locally & on jenkins I believe) https://jenkins.status.im/job/status-go/job/status-go-manual/541/console

Reply:
I was not aware of `yes` command O_o it keeps printing `y` in new lines.

Reply:
Dude, you've missed out. Sometimes when I just don't know what to do anymore I just type `yes wtf` into the console.